### PR TITLE
feat: add bandit as defined-formats

### PR DESCRIPTION
**Pull request by Alt-NoRock**
defined bandit as defined-formats:
- add bandit errorformat to python.go
- add test-pattern
- generated dog.go referenced by fmt/README.md

### CODE REVIEW

**Comment by Alt-NoRock** [comment]
In locally test, I fixed #128 first.

File: fmts/testdata/bandit.in
Comment (shogo82148) [inline fmts/testdata/bandit.in:1]: This output is in a human-readable format.
It is better to use [the custom formatter](https://bandit.readthedocs.io/en/latest/formatters/custom.html) and pass it to errorformat.

Comment (haya14busa) [reply fmts/testdata/bandit.in:1]: While I definitely agree that it's better to use custom formatter and output rdjson or use json formatter, I'm fine with including this change for handy use for now. 

WDYT?

**Review by haya14busa — APPROVED** [review]

**Review by shogo82148 — APPROVED** [review]
no problem

**Comment by Alt-NoRock** [comment]
Thank you for your comments.
According to your review, I will use custom formatter and output in the next PR.

And I rebased this branch to resolve conflict.

File: fmts/testdata/bandit.in
Comment (Alt-NoRock) [reply fmts/testdata/bandit.in:1]: I changed format  human-readable to [the custom formatter](https://bandit.readthedocs.io/en/latest/formatters/custom.html) in 72b569dcf08dee0c27fe9c100502d8c8769b7815

**Comment by Alt-NoRock** [comment]
Sorry to late.
I changed defined format as your review.
Can you check this PR?

File: fmts/testdata/bandit.in
Comment (shogo82148) [reply fmts/testdata/bandit.in:1]: Thanks!

**Review by shogo82148 — APPROVED** [review]
LGTM

**Pull request merged**